config/prod.json

```diff
@@ -0,0 +1,21 @@
+{
+  "server": {
+    "port": 80,
+    "debug": false
+  },
+  "folder": {
+    "public": "/public",
+    "live": "/video/live",
+    "replay": "/video/replay",
+    "save": "/video/save",
+    "assets": "/assets"
+  },
+  "stream": {
+    "width": 1280,
```

Comment:
wouldn't be better to just add a `command` entry here (and maybe `arguments`), if we want to change the recorder easily between envs or in the future ?

Reply:
We have to split the command since we add `--output` (or equivalent depending of used library) argument. But we can refactor like that : 
```json
{
  "stream": {
    "module": "raspivid_or_any_lib_in_src/stream_folder",
    "arguments": {
      "--width": 1280,
      ...
    }
  }
}
```